Gems/PhysX/Code/Source/EditorShapeColliderComponent.cpp

```diff
@@ -354,6 +381,60 @@ namespace PhysX
         m_geometryCache.m_boxDimensions = scale * boxDimensions;
     }
 
+    void EditorShapeColliderComponent::UpdateQuadConfig(const AZ::Vector3& scale)
+    {
+        LmbrCentral::QuadShapeConfig quadShapeConfig;
+        LmbrCentral::QuadShapeComponentRequestBus::EventResult(quadShapeConfig, GetEntityId(),
+            &LmbrCentral::QuadShapeComponentRequests::GetQuadConfiguration);
+
+        const float minDimension = 1e-3f; // used to prevent the dimensions being 0 in any direction
+        const float xDim = AZ::GetMax(minDimension, quadShapeConfig.m_width);
+        const float yDim = AZ::GetMax(minDimension, quadShapeConfig.m_height);
+
+        if (m_singleSided)
+        {
+            AZStd::vector<AZ::u8> cookedData;
+
+            constexpr AZ::u32 vertexCount = 4;
+            const AZ::Vector3 vertices[vertexCount] =
```

Comment:
suggestion: Could upgrade to `AZStd::array` instead to get some of the nicer API additions 🙂

Reply:
I think the usage is so simple here that the current approach is fine.